.github/workflows/ci-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: Test ASReview webapp (integration)
+on: [push, pull_request]
+permissions:
+  contents: read
+jobs:
+  selenium-tests:
+    env:
+      ASREVIEW_LAB_CONFIG_PATH: asreview/webapp/tests/config/auth_integration.toml
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-depth: 0
+          fetch-tags: true
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20"
+          cache: "npm"
+          cache-dependency-path: "asreview/webapp/package-lock.json"
+      - uses: nanasess/setup-chromedriver@v2
+      - name: Install the necessary packages
+        run: pip3 install .[test] setuptools gunicorn
+      - name: Compile assets
+        run: python setup.py compile_assets
+      - name: Start ASReview
+        run: |
+          pip3 install .
+      - name: Run tests
+        run: |
+          export DISPLAY=:99
+          asreview lab --no-browser --config-path asreview/webapp/tests/config/auth_integration.toml --skip-update-check &
```

Comment:
The tests fail because this line seems to immediately exit, so there is no server running to test against.

Weirdly, I get the same result locally when I run:

`asreview lab --config-path asreview/webapp/tests/config/auth_integration.toml`

When I leave out `--config-path`, it works. Any idea what might be going on here?

Reply:
This was due to setting `Testing = true` in the app config. Not sure if this is indeed desired behavior, but I'll close this for now.